app/main.ipynb

```diff
@@ -192,16 +247,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
+   "execution_count": 28,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "array([3.5, 5. , 7. , 6.5, 4. ])"
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your code here"
+    "A.mean(axis=1)"
```

Comment:
incorrect result, you have 2 columns, but got 5 values, weird, isn't it?)
fix everywhere, here is a tip:
![image](https://github.com/mate-academy/py-numpy/assets/61380547/702f673e-513a-4f60-94f3-e606cb03cece)

Reply:
thanks)